src/javascript/binary/pages.js

```diff
@@ -30,7 +30,11 @@ var trading_times_init = function() {
                   success: function(html){
                             trading_times.replaceWith(html);
                             trading_times = $("#trading-tabs");
-                            trading_times.tabs();
+			    if (page.language() === 'JA') {	
```

Comment:
@borisyankov @jy-binary I tried adding a class in the html to hide tabs (something like <li [% IF market.tabid == 'forex' %] class='hide-ja' [% END %]), but without the javascript modification it doesn't work. this is the solution that raunak suggested and it seems to be the only thing that's working.